docker-compose.yaml

```diff
@@ -1,87 +1,35 @@
 version: "3.8"
 
 services:
-  prod:
-    build:
-      context: .
-      dockerfile: ./compose/waitress/Dockerfile
-    volumes:
-      - .:/app
-    ports:
-      - "8090:8080"
-    env_file:
-      - .env.shared
-    depends_on:
-      - postgresql
-      - celery-worker
-
-  dev:
-    build:
-      context: .
-      dockerfile: ./compose/flask/Dockerfile
-    volumes:
-      - .:/app
-    ports:
-      - 5010:5000
-    env_file:
-      - .env.shared
-    environment:
-      - GRAFANA_HOST=grafana-dev
-    depends_on:
-      - postgresql
-      - celery-worker
-      - grafana-dev
-
   postgresql:
```

Comment:
That means we don't use the optimized version of the Postgres database in the tests anymore, is that correct? 😇

Reply:
we still do, if you have a look at docker-compose.test.yaml all the parameters from postgres-test are overridden with this compose file

Reply:
Sorry and thanks 😇 It's already late :P